src/app/core/services/notification.service.ts

```diff
@@ -261,7 +254,8 @@ export class NotificationService {
     const task = data.task
     const scheduledTimestamp = task.timestamp
     const now = new Date().getTime()
-    const endScheduledTimestamp = scheduledTimestamp + 1000 * 60 * 10
+    const endScheduledTimestamp =
```

Comment:
would be good to document that endtime is 10 min later than scheduled time and why

Reply:
I've fixed it to use the completion window value instead since this is already available.